components/BlogPostCard.js

```diff
@@ -0,0 +1,89 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+import blogPostShape from '../propTypes/blogPost'
+import { RouteNames } from '../routes'
+import { extraSmallSpacing, smallSpacing } from '../styling/sizes'
+import { footerText } from '../styling/typography'
+import { formatDateForLocale } from '../utils/locale'
+import Link from './Link'
+
+const Card = styled.a`
+  background-color: ${({ theme }) => theme.orangeBackgroundMedium};
+  color: inherit;
+  text-decoration: none;
+  transition: background-color .15s ease-in-out;
+
+  display: flex;
+  flex-direction: column;
+
+  &:hover, &:focus {
+    background-color: ${({ theme }) => theme.orangeBackgroundDark};
+    color: inherit;
+    text-decoration: none;
+  }
+`
+
+const ImageContainer = styled.div`
+  background-color: ${({ theme }) => theme.orangeBackgroundDark};
+  height: 200px;
+  overflow: hidden;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`
+
+const Image = styled.img`
+  width: 100%;
+`
+
+const CardContent = styled.div`
+  padding: ${smallSpacing};
+`
+
+const Metadata = styled.p`
+  ${footerText};
+  margin: 0;
+`
+const Title = styled.h6`
+  font-size: 22px;
+  font-weight: 800;
+  margin: ${extraSmallSpacing} 0 0 0;
+`
+
+const TeaserText = styled.p`
+  margin: ${smallSpacing} 0 0 0;
+`
+
+const BlogPostCard = ({ post, locale }) => (
+  // ESLint seems to think that this link misses a href. It doesn't.
+  // eslint-disable-next-line jsx-a11y/anchor-is-valid
```

Comment:
Erik was suggesting some time ago to disable this rule completely. Maybe we should do it?

Reply:
Yeah I think so too, actually. We're suppressing this rule in almost every file.